src/main/java/org/openhab/binding/melcloud/internal/handler/MelCloudDeviceHandler.java

```diff
@@ -122,6 +122,7 @@ private void initializeBridge(ThingHandler thingHandler, ThingStatus bridgeStatu
 
             if (bridgeStatus == ThingStatus.ONLINE) {
                 updateStatus(ThingStatus.ONLINE);
+                startAutomaticRefresh();
```

Comment:
This shouldn't be needed as bridgeStatusChanged is called by the framework which start the refresh task

Reply:
I can't say for sure what the source of the issue was/is but only that I could reproduce it every time without calling startAutomaticRefresh at this point.
I'm not entirely happy by calling it an additional time here but there is a check in startAutomaticRefresh that will prevent the code to be run twice if not needed and it does fix or work around the issue.

If you know the  source of the issue or a better way to handle this I'd love to hear it as that way I can learn more about the framework part :)